src/app/learn/get-started/basics/modules/index.html

```diff
@@ -53,12 +53,6 @@
   </stache-page-anchor>
 
   <p>
-    To update the packages and modules that you install as dependencies in your project, run <sky-code>skyux upgrade</sky-code>.
-  </p>
-  <sky-alert alertType=info>
-    The <sky-code>skyux upgrade</sky-code> command currently requires you to install <sky-code>@skyux-sdk/builder-plugin-migrate</sky-code> globally. We are exploring moving this command to the main SKY UX CLI package. For more information, see the <a href="https://developer.blackbaud.com/skyux-migration-guide">SKY UX migration guide</a>.
-  </sky-alert>
-  <p>
-    This command pulls in the latest version of the <sky-code>devDependencies</sky-code> and <sky-code>peerDependencies</sky-code> as well as any dependencies listed in the SPA's <sky-code>package.json</sky-code>.
+    To update the packages and modules that you install as dependencies in your project, run <sky-code>skyux upgrade</sky-code>. This command pulls in the latest minor versions or patches of <sky-code>devDependencies</sky-code> and <sky-code>peerDependencies</sky-code> as well as any dependencies listed in the SPA's <sky-code>package.json</sky-code>.
```

Comment:
I think we could pare this down a bit, as even the original text is pretty confusing. I don't understand the distinction between "`devDependencies` and `peerDependencies`" and "dependencies listed in the SPA's `package.json`", and mentioning `peerDependencies` is confusing and misleading since we don't touch them during `skyux upgrade`. This would be a more accurate description:

`To update your project's dependencies, run <sky-code>skyux upgrade</sky-code>. This command installs the latest minor and patch versions of packages listed in your <sky-code>package.json</sky-code> file's <sky-code>dependencies</sky-code> and <sky-code>dependencies</sky-code> and updates the references listed in that file.`

Reply:
I assume the second `dependencies` in your update should be `devDependencies`. I'll make the updates with that assumption, but let me know if I'm wrong.